malasakit-django/cafe/settings.py

```diff
@@ -1,182 +0,0 @@
-"""
```

Comment:
Put this settings file back. The secret key is no longer there, so there's no reason to remove this file.